xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h

```diff
@@ -180,6 +180,11 @@ extern "C" {
    */
   const int PVR_CHANNEL_INVALID_UID = -1; /*!< @brief denotes that no channel uid is available. */
 
+  /*!
+   * @brief special PVR_RECORDING.iRecordingState value to indicate that no recording state is available.
+   */
+  const int PVR_RECORDED_TV_ITEM_STATE_INVALID = -1; /*!< @brief denotes that no recording state is available. */
```

Comment:
PVR_RECORDED_TV_ITEM_STATE_INVALID = 0, please.

PVR_RECORDED_TV_ITEM_STATE_INVALID must not happen! We must force the addon devs to support this property. If we define it as 0 we catch addons not even writing PVR_RECORDED_TV_ITEM_STATE_INVALID in the code, becasue default init for PVR_RECORDING is just zeroing out everything. ;-)